x-pack/solutions/observability/plugins/slo/server/lib/rules/slo_burn_rate/executor.ts

```diff
@@ -168,13 +168,13 @@ export const getRuleExecutor = (basePath: IBasePath) =>
             actionGroup,
             state: {
               alertState: AlertStates.ALERT,
-              grouping: groupings,
             },
             payload: {
               [ALERT_REASON]: reason,
               [ALERT_EVALUATION_THRESHOLD]: windowDef.burnRateThreshold,
               [ALERT_EVALUATION_VALUE]: Math.min(longWindowBurnRate, shortWindowBurnRate),
               [ALERT_GROUP]: groups,
+              [ALERT_GROUPING]: groupings, // Object, example: { host: { name: 'host-0' } }
               [SLO_ID_FIELD]: slo.id,
               [SLO_REVISION_FIELD]: slo.revision,
               [SLO_INSTANCE_ID_FIELD]: instanceId,
```

Comment:
@benakansara Regarding our discussion for the `...getEcsGroups(groups)`, you mentioned I should use grouping object instead of groups array as it can give false positives. Since I changed the way groups is constructed, do I need to do any more change here?

I can see the ECS field stored in the alerts as data indices:

<img width="368" height="37" alt="Screenshot 2025-07-22 at 18 11 46" src="https://github.com/user-attachments/assets/06c78be6-c3f9-41e5-98a7-bf70a355fe1a" />